pyccel/parser/semantic.py

```diff
@@ -2143,13 +1880,11 @@ def _visit_Assign(self, expr, **settings):
             stmts[-1] = stmt
             return CodeBlock(stmts)
 
-        elif isinstance(rhs, Application):
+        elif isinstance(rhs, FunctionCall):
 
             # ARA: needed for functions defined only with a header
```

Comment:
Is this comment still accurate? If not we should modify it or remove it.

Reply:
no